packages-next/fields/src/types/relationship/views/cards/index.tsx

```diff
@@ -123,125 +133,150 @@ export function Cards({
-                    });
-                  }}
-                  tone="active"
-                >
-                  Edit
-                </Button>
-              )}
-              {field.display.removeMode === 'disconnect' && onChange !== undefined && (
-                <Tooltip content="This item will not be deleted. It will only be removed from this field.">
-                  {props => (
-                    <Button
-                      size="small"
-                      disabled={onChange === undefined}
-                      onClick={() => {
-                        const currentIds = new Set(value.currentIds);
-                        currentIds.delete(id);
-                        onChange({
-                          ...value,
-                          currentIds,
-                        });
-                      }}
-                      {...props}
-                      tone="negative"
-                    >
-                      Remove
-                    </Button>
-                  )}
-                </Tooltip>
-              )}
-              {field.display.linkToItem && (
-                <Button
-                  size="small"
-                  weight="link"
-                  tone="active"
-                  css={{ textDecoration: 'none' }}
-                  as={Link}
-                  href={`/${foreignList.path}/${id}`}
-                >
-                  View {foreignList.singular} details
-                </Button>
-              )}
-            </Stack>
-          </CardContainer>
-        );
-      })}
+          return value.itemsBeingEdited.has(id) && onChange !== undefined ? (
+            <CardContainer mode="edit" key={id}>
+              <VisuallyHidden as="h2">{`${field.label} ${index + 1}: Edit mode`}</VisuallyHidden>
+              <InlineEdit
+                list={foreignList}
+                fields={field.display.inlineEdit!.fields}
+                onSave={newItemGetter => {
+                  setItems({
+                    ...items,
+                    [id]: newItemGetter,
+                  });
+                  const itemsBeingEdited = new Set(value.itemsBeingEdited);
+                  itemsBeingEdited.delete(id);
+                  onChange({
+                    ...value,
+                    itemsBeingEdited,
+                  });
+                }}
+                selectedFields={selectedFields}
+                itemGetter={itemGetter}
+                onCancel={() => {
+                  const itemsBeingEdited = new Set(value.itemsBeingEdited);
+                  itemsBeingEdited.delete(id);
+                  onChange({
+                    ...value,
+                    itemsBeingEdited,
+                  });
+                }}
+              />
+            </CardContainer>
+          ) : (
+            <CardContainer mode="view" key={id}>
+              <VisuallyHidden as="h2">{`${field.label} ${index + 1}`}</VisuallyHidden>
```

Comment:
Adding a visually hidden heading to demarcate and differentiate list item content to screen readers.